### DISCUSSION TITLE
SSO with AZURE AD (Cloud Tenant)

### DISCUSSION THREAD

**Discussion by dalicea** [snipe/snipe-it#9855] [Q&A] [answered]
A question about the Hosted version.

Does the hosted allow for easier integration of SSO with AZURE AD (Cloud Tenant)?  We are having issues trying to configure SSO with AZURE per the procedures provided in the manual.

Thanks,
Dave

**Comment by AndyNDuffy** [comment]
Hi Dave
I am a really new user of Snipe-IT but we do have the cloud version. I am hoping to integrate it with our Azure AD so will let you know how it goes. What documentation have you been following?

**Comment by dalicea** [comment, marked as answer]
We were able to connect our on-premise instance to AZURE AD (cloud tenant) using the instructions provided in this thread https://github.com/snipe/snipe-it/issues/9651#issuecomment-852939823.  The one quirk with SAML to AZURE AD, is that the user needs to exist in SNIPE-IT in order to validate the AZURE AD account.  So as I see it, this configuration is not a true SSO integration.

Hope this helps!

Dave

**Comment by AndyNDuffy** [comment]
@dalicea I know its a bit of a fudge, but I was thinking of using Power Automate to connect to their API and add any new users.

**Reply by dalicea** [reply]
I haven't tried automating user account updates.  Hopefully, it works out for you.
